security/crunch/Portfile

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
 PortSystem      1.0
+PortGroup       makefile 1.0
 
 name            crunch
 version         3.6
-revision        0
+revision        1
```

Comment:
I don't think there is a reason to increase the revision here, but please double-check the e-mail that Ryan send about this recently

Reply:
Using the makefile portgroup might cause MacPorts optimization flags (and other flags that might change installed files) to be used now when they weren't before. You can do before and after builds and compare the logs to see if anything's different (easier if you use `build.jobs=1`)